src/components/CameraControls/index.tsx

```diff
@@ -141,7 +144,21 @@ const CameraControls = ({
         }
         lastKeyPressed.current = keyPressed;
     }, [keyPressed]);
-    return (
+
+    const homeViewButton = (
```

Comment:
In my PR I factored this out into a component within CameraControls so you don't have it copied in the PlaybackControls. Is there a reason you didn't keep that change?

Reply:
I merged in your PR and don't think I changed that in my small tweaks to your PR, so not sure why it didn't come along for the ride.

Reply:
Not seeing it in here, am I missing something? https://github.com/simularium/simularium-website/pull/568

Reply:
I forgot to push it! I opened a new PR, the histories are now a little messed up so I wouldn't try to merge it, I would just look at it as the suggestion

Reply:
https://github.com/simularium/simularium-website/pull/571